### PR TITLE
Full library setup

### PR DESCRIPTION
**Pull request by a-urth**
Tox, readme, requirements, tests, travis etc.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+.. code-block:: python
+
+    import asyncio
+
+    from aioneo4j import Neo4j
+
+
+    async def go():
+        url = 'http://neo4j:neo4j@127.0.0.1:7474/'
```
Comment (hellysmile) [inline README.rst:30]: url not used

File: aioneo4j/client.py [hunk 2]
```diff
@@ -176,15 +176,14 @@ def user_password(
...
-        yield from self.transport.close()
+        return self.transport.close()
 
-    if PY_350:
+    if PY_352:
         @asyncio.coroutine
         def __aenter__(self):  # noqa
             return self
 
         @asyncio.coroutine
```
Comment (hellysmile) [inline aioneo4j/client.py]: extra coroutine

File: aioneo4j/transport.py [hunk 3]
```diff
@@ -174,7 +174,10 @@ def perform_request(
 
         return status, data
 
-    @asyncio.coroutine
     def close(self):
         if self.session is not None:
```
Comment (hellysmile) [inline aioneo4j/transport.py]: remove lazy session at all

**Review by hellysmile — CHANGES_REQUESTED** [review]

File: .travis.yml [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+addons:
+  apt:
+    packages:
+      - oracle-java8-set-default
+python:
+  - "3.4"
+  - "3.5"
+  - "3.6"
+env:
+  - NEO4J="3.1.3"
```
Comment (hellysmile) [inline .travis.yml]: 3.0.? Please

File: requirements.txt [hunk 5]
```diff
@@ -0,0 +1,17 @@
+aiohttp==2.0.5
```
Comment (hellysmile) [inline requirements.txt:1]: Copy all dec stuff from aioeladticsearch please

File: tests/conftest.py [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+
+
+@pytest.fixture
+def neo4j(loop):
+    neo4j = Neo4j(loop=loop)
+    neo4j.auth = ('neo4j', 'neo4jneo4j')
+
+    try:
+        yield neo4j
+    finally:
```
Comment (hellysmile) [inline tests/conftest.py:33]: Drop before and after evrything from neo4j

**Review by hellysmile — COMMENTED** [review]
Can You give a list of missing rest endpoints except transaction related?

File: README.rst (hunk 1 above)
Comment (hellysmile) [reply README.rst:30]: Fix this please

File: tests/conftest.py (hunk 6 above)
Comment (a-urth) [reply tests/conftest.py:33]: this is very bad idea to not to mock actual neo4j and drop everything from it.
if someone runs local neo4j and he runs tests he'll loose all data.

File: requirements.txt (hunk 5 above)
Comment (a-urth) [reply requirements.txt:1]: copy what? and why? here are all required deps

**Pull request merged**